test/e2e/mount_test.go

```diff
@@ -136,19 +137,38 @@ func setupTestSuite() {
 	gcloudCmd.Env = append(os.Environ(), "KUBECONFIG="+f.kubeconfigFile)
 	check(execCmd(gcloudCmd))
 
-	// Install Secret Store
-	check(execCmd(exec.Command("kubectl", "apply", "--kubeconfig", f.kubeconfigFile,
-		"-f", fmt.Sprintf("https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/%s/deploy/rbac-secretproviderclass.yaml", f.secretStoreVersion),
-		"-f", fmt.Sprintf("https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/%s/deploy/rbac-secretprovidersyncing.yaml", f.secretStoreVersion),
-		"-f", fmt.Sprintf("https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/%s/deploy/csidriver.yaml", f.secretStoreVersion),
-		"-f", fmt.Sprintf("https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/%s/deploy/secrets-store.csi.x-k8s.io_secretproviderclasses.yaml", f.secretStoreVersion),
-		"-f", fmt.Sprintf("https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/%s/deploy/secrets-store.csi.x-k8s.io_secretproviderclasspodstatuses.yaml", f.secretStoreVersion),
-		"-f", fmt.Sprintf("https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/%s/deploy/secrets-store-csi-driver.yaml", f.secretStoreVersion),
-	)))
+	// Helm authentication
+	gcloudCmd := exec.Command("gcloud", "auth", "application-default", "print-access-token")
+
+	// Capture the output of the gcloud command
+	gcloudOutput, err := gcloudCmd.Output()
+	
+	if err != nil {
+		fmt.Printf("Error running gcloud command: %s\n", err.Error())
+		os.Exit(1)
+	}
+	// Create the helm registry login command
+	helmCmd := exec.Command("helm", "registry", "login", "-u", "oauth2accesstoken", "--password-stdin", "https://asia-east1-docker.pkg.dev")
```

Comment:
Can you confirm if the usage of asia-east1 across this commit is just a choice of region & it can be substituted with a different supported region if required?
If yes, can this be made a constant which is referred everywhere?